tests/nat/builder/test_builder.py

```
@@ -1665,6 +1666,16 @@ def __init__(self, raise_error: bool = False):
             super().__init__()
             self.raise_error = raise_error
 
+        @property
+        def enabled(self) -> bool:
+            return True
+
+        async def pre_invoke(self, context):
```

Ruff (0.14.8)

Unused method argument: `context`

(ARG002)